### ISSUE TITLE
Rich Presence SDK has been deprecated in favor of "GameSDK"

### ISSUE THREAD

**Issue by FichteFoll** [FichteFoll/discordrp-mpris#13]
The Discord docs point towards their rich presence SDK being deprecated in favor of their GameSDK's [ActivityManager](https://discordapp.com/developers/docs/game-sdk/activities). However, from a brief look that seems to rely on games being approved by staff, which is even paused at the moment. Since we're not wrapping a game here, I'd like to investigate what this change of suggestion means for this tool and whether something needs to be changed or whether this is just about the SDK and the underlying protocol remains unchanged.

**Comment by bbigras** [comment]
Any update on this?

**Comment by FichteFoll** [comment]
No, this is just a todo item far back in the log. Basically every other issue here is more important (to me, at least).

**Comment by bbigras** [comment]
ok no hurries. I was just wondering if this change could kill the project. thanks

**Comment by FichteFoll** [comment]
I personally don't forsee Discord deprecating this endpoint in the near future, considering just how many tools already adopted it before they added the ActivityManager thing. Even if they did, I don't forsee them closing the gates on all the various third-party tools that use their endpoint to show non-game data entirely.

In the end, I highly doubt this will kill the project, if that's all you were interested in.

**Comment by FichteFoll** [comment]
It appears that recently an ActivityType enum has been added that allows setting the activity to "Listening" (via https://discord.com/developers/docs/developer-tools/game-sdk#activitytype-enum, https://github.com/qwertyquerty/pypresence/pull/244).
